opendevin/controller/agent_controller.py

```diff
@@ -65,6 +67,7 @@ def __init__(
             sid: The session ID of the agent.
             max_iterations: The maximum number of iterations the agent can run.
             max_chars: The maximum number of characters the agent can output.
+            max_budget_per_task: The maximum budget allowed per task, beyond which the agent will stop.
```

Comment:
Nit: could we mention the unit of this config?
```suggestion
            max_budget_per_task: The maximum budget (in USD) allowed per task, beyond which the agent will stop.
```